packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx

```diff
@@ -85,12 +92,18 @@ const motionDivProps = {
 export const Experimental = () => {
     const enabledFeatures = useSelector(state => state.suite.settings.experimental);
     const isDebug = useSelector(selectIsDebugModeActive);
+    const { translationString } = useTranslation();
+    const state = useSelector(state => state);
```

Comment:
Now it's almost what I wanted in [that comment](https://github.com/trezor/trezor-suite/pull/20308/files#r2240107192), now there is no need anymore to select whole state in component :slightly_smiling_face: 
```suggestion
    const isBioAuthAvailable = useSelector(selectIsBioAuthAvailable);
```